CMakeLists.txt

```
@@ -202,6 +204,13 @@ install(
 foreach (FILE README.asciidoc COPYING ${GENERATED_DOC_FILES})
     install(FILES "${FILE}" DESTINATION "${OS_AUTOINST_DOC_DIR}")
 endforeach ()
+foreach (PYTHON_BINDING IN LISTS PYTHON_BINDINGS)
+    configure_file("api.py.in" "${PYTHON_BINDING}.py")
```

We already have many files at the root. Maybe this is a chance to tidy that up a little bit, e.g.:
```suggestion
    configure_file("cmake/templates/api.py.in" "${PYTHON_BINDING}.py")
```
(And of course move the actual file.)

---

:+1: